boa/src/builtins/value/mod.rs

```diff
@@ -43,7 +43,7 @@ pub enum ValueData {
     /// `String` - A UTF-8 string, such as `"Hello, world"`
     String(String),
     /// `Number` - A 64-bit floating point number, such as `3.1415`
-    Number(f64),
+    Racional(f64),
```

Comment:
Why is this `Racional` instead of `Rational`?

Reply:
> Why is this `Racional` instead of `Rational`?

Good catch! I wouldn't have noticed if I wasn't concentrated on it. It's because in my native language that's
how you spell `Rational` in my mind it didn't seem wrong at the time.

You're right it should be `Rational`. :)

Reply:
It happens to me too. In Spanish it's spelled "racional" too haha